src/main/java/it/gov/pagopa/pu/organization/connector/taxonomy/client/TaxonomySyncClient.java

```diff
@@ -0,0 +1,28 @@
+package it.gov.pagopa.pu.organization.connector.taxonomy.client;
+
+import it.gov.pagopa.pu.organization.connector.taxonomy.config.TaxonomyApisHolder;
+import it.gov.pagopa.pu.pagopapayments.dto.generated.Taxonomy;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Lazy
```

Comment:
@Lazy is to be used only inside activities, we can remove it outside that library